judges/github-events/github-events.rb

```diff
@@ -36,6 +36,25 @@ def self.skip_event(json)
     raise Factbase::Rollback
   end
 
+  def self.ignored_bot?(fact)
+    return false unless fact.all_properties.include?('who')
+    return false unless $options.respond_to?(:bots)
+    return false if $options.bots.nil? || $options.bots.empty?
+    begin
+      user = Fbe.octo.user(fact.who)
+      login = user[:login]
+      ignored_bots = $options.bots.split(',').map(&:strip)
```

Comment:
@danilax86 @yegor256 Perhaps it's worth caching this action (`$options.bots.split(',').map(&:strip)`) somehow.
I'm not sure, but maybe it's add this array to `$locals` or use instance var.